conf/script/build-ha-io

```diff
@@ -517,6 +524,16 @@ motr_kernel_rsc_add() {
                                            kind=Optional symmetrical=false
 }
 
+var_motr_rsc_add() {
+    log "${FUNCNAME[0]}: Adding var-motr mounts to Pacemaker..."
+    sudo pcs -f $cib_file resource create var-motr1 ocf:heartbeat:Filesystem \
```

Comment:
need to add this resource to cleanup as well, prov-ha-reset script.

Reply:
I think it will be cleaned as part of c1/2 group there (the same was as hax and others).

Reply:
ah  right!